transcom/mymove
# B-19573 made changes to remove commas before sending to the validator function

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19573)

## Summary
When creating a shipment as a service counselor, this field would attempt to validate the input which was decorated with a comma, such as ```12,000```

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Log in as a service counselor, and create a shipment for a move.
2. Under the estimated weight section, enter in a weight which would be given a comma delimited value.
3. Swap a number by highlighting the characters to switch, and overwrite the digit.
4. A validation error would pop up like this:
<img width="299" alt="image" src="https://github.com/transcom/mymove/assets/148263228/3350ed7d-5d27-4eed-9a1c-4762fd193f82">
(This is due to the comma being provided to the event pipeline, and the parsing functions within the validator is unable to cast a non-number string to a number).
But instead, we see the message does not pop up

### Frontend


### Backend



## Screenshots
